includes/definitions/discovery/querx.yaml

```diff
@@ -0,0 +1,18 @@
+mib: EGNITEPHYSENSOR-MIB
+modules:
+    sensors:
+        temperature:
+            data:
+                -
+                    oid: querxPhySensorTable
+                    value: querxPhySensorValue
+                    num_oid: '.1.3.6.1.4.1.3444.1.14.1.2.1.5.1'
```

Comment:
Need to change this to a more dynamic structure, and include `$index`in the num_oid, see comments.